types/utils.d.ts

```diff
@@ -0,0 +1,5 @@
+// ========================================================
+// Props
+// ========================================================
+
+export type ItemShorthand = React.ReactNode | object | (React.ReactNode | object)[]
```

Comment:
Will put it in the Button PR, as there I will finally have all components so that I can update them accordiongly. That is why I introduced just this type here.

Reply:
got it, thanks!